### PR TITLE
Add operator object storage docs

### PR DESCRIPTION
**Pull request by pavolloffay**
**What this PR does**:

Follow up after https://github.com/grafana/tempo/pull/2596 

The PR migrates content from https://tempo-operator.netlify.app/docs/object_storage.md/ to Grafana docs

**Which issue(s) this PR fixes**:

**Checklist**
- [ ] Tests updated
- [x ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Comment by pavolloffay** [comment]
@knylander-grafana  could you please review?

File: docs/sources/tempo/setup/operator/object-storage.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
+---
+title: 'Object storage'
+description: Setup for storing traces to Object Storage
+menuTitle: Object storage
+weight: 200
+aliases:
+- /docs/tempo/operator/object-storage
+---
+
```
Comment (knylander-grafana) [inline docs/sources/tempo/setup/operator/object-storage.md:9]: I'm adding a page title that matches the title set in the front matter. 
```suggestion

# Object storage for Operator
```

File: docs/sources/tempo/setup/operator/object-storage.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
+---
+title: 'Object storage'
```
Comment (knylander-grafana) [inline docs/sources/tempo/setup/operator/object-storage.md:2]: The single quotes aren't needed around a title in front matter. 
```suggestion
title: Object storage
```

File: docs/sources/tempo/setup/operator/object-storage.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+  ```yaml
+  spec:
+    storage:
+      secret:
+        name: tempostack-dev-gcs
+        type: gcs
+  ```
+
+## Minio
```
Comment (knylander-grafana) [inline docs/sources/tempo/setup/operator/object-storage.md:109]: According to the MinIO website (linked below), the capitalization should be MinIO. 
```suggestion
## MinIO
```

File: docs/sources/tempo/setup/operator/object-storage.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+### Requirements
+
+* Deploy Minio on your Cluster, e.g. using the [Minio Operator](https://operator.min.io/)
+
+* Create a [bucket](https://docs.min.io/docs/minio-client-complete-guide.html) on Minio via CLI.
```
Comment (knylander-grafana) [inline docs/sources/tempo/setup/operator/object-storage.md:115]:
```suggestion
* Deploy MinIO on your cluster using the [Minio Operator](https://operator.min.io/) or another method. 

* Create a [bucket](https://docs.min.io/docs/minio-client-complete-guide.html) on MinIO via CLI.
```

**Comment by pavolloffay** [comment]
@knylander-grafana I have fixed the comments

**Review by knylander-grafana — APPROVED** [review]
Thank you for adding docs.

**Pull request merged**